components/dropdown/user/src/index.js

```diff
@@ -30,27 +30,28 @@ class DropdownUser extends Component {
     })
   }
 
-  _renderLink = ({ text, url, icon: Icon }, index) => {
+  _renderLink = ({ text, url, icon: Icon, notifications }, index) => {
     const Link = this.props.linkFactory
 
     return (
       <li key={index} className='sui-DropdownUserMenu-listItem'>
         <Link href={url} className='sui-DropdownUserMenu-listLink' title={text}>
           <Icon svgClass='sui-DropdownUserMenu-listIcon' />
           <span>{text}</span>
+          {!!notifications && <span className='sui-DropdownUserMenu-listNotification'>{notifications}</span>}
         </Link>
       </li>
     )
   }
 
   render () {
     const { expanded, collapseByTouch } = this.state
-    const { user, menu, expandOnMouseOver } = this.props
+    const { user, menu, expandOnMouseOver, hasNotifications } = this.props
```

Comment:
Better optionally assign a value based on menu notifications: `const { user, menu, expandOnMouseOver, hasNotifications = this.props.menu.some(({ notifications }) => Boolean(notifications) } = this.props`